src/App.tsx

```diff
@@ -1,14 +1,53 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
I'm writing this because I couldn't find anything wrong in the code 😄 

You can import only FC here

```ts
import { FC, useEffect, useState } from 'react';
```

Then you can use it like

```ts
export const App: FC = () => {...
```
instead of 
```ts
export const App: React.FC = () => {
```